src/server/handleRequest.js

```diff
@@ -100,7 +100,7 @@ function renderFullPage(html, initialState, head) {
         <meta name="msapplication-TileImage" content="/static/icons/ms-icon-144x144.png" />
         <meta name="theme-color" content="#ffffff" />
       </head>
-      <body>
+      <body style="margin: 0;">
```

Comment:
ID instead of inline CSS?

Reply:
This is to make the title bar look good while the page is loading before the CSS bundle is generated. Unrelated to this PR, but I was tired of it stuttering while loading.